### ISSUE TITLE
res.cookie is not a function

### ISSUE THREAD

**Issue by zakaria-chahboun** [lukeed/polka#144]
I want to use the package `cookie-parser`
BUT
I can't use cookie function in res in my middleware! it gives me an error message `res.cookie is not a function`

my code look like this:

```js
import cookieParser from "cookie-parser";
```

```js
polka()
	// - body parser
	.use(json())
	// - cookies parser
	.use(cookieParser())
	.all("*", (req, res, next) => {
		res.cookie("XSRF-TOKEN", req.csrfToken());
		next();
	})
```

**Comment by lukeed** [comment]
Correct, it is not. You have to set it up yourself (until there's an official package, which is definitely planned). 

Until then, copying the `cookie` example works great: https://github.com/lukeed/polka/issues/83#issuecomment-455289371

**Comment by zakaria-chahboun** [comment]
@lukeed 
i did this 
```js
.get('/api/v1/login', (req,res,next)=>{
...
const expiresIn = 60 * 60 * 24 * 5 * 1000;
res.setHeader('Set-Cookie', cookie.serialize("session", {maxAge: expiresIn});
...
}
)
```
Now in the browser i can see my `session` cookie, but the propriety `path` is `/api/v1`
![image](https://user-images.githubusercontent.com/18173275/92153156-b8317c80-ee1b-11ea-8397-b80b8c4b43db.png)

So, this session is valid only in the routes with `api/v1` path!
How i can set this path to `/` for global access?
Because the problem is even if the user logs in, i cant access to this session from for home page '/' (for example) and so on!
